CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
```

Comment:
Please also add a contribution-welcome section in the main README file linking to this one.

Reply:
Added.